### DISCUSSION TITLE
There are many logs about `evbug` in dmegs

### DISCUSSION THREAD

**Discussion by EastWoodYang** [devmfc/debian-on-amlogic#49] [Q&A]
HI~ @devmfc There are many logs about `evbug` in dmegs, do you have any ideas about this.

![image](https://github.com/devmfc/debian-on-amlogic/assets/9335046/3a90dbf2-da42-4e88-89e6-9237eac557a7)

![image](https://github.com/devmfc/debian-on-amlogic/assets/9335046/409ef202-fdad-4252-bc7a-66f1837c8055)

**Comment by devmfc** [comment]
Seems to me like a input device connected to your box. Mouse/keyboard or remote?

**Reply by EastWoodYang** [reply]
Yes, keyboard... can you turn this log off ?
